isM100: true


rcVelocityMultiplier: .3


hVelocityMaxBody: 3.0
hVelocityMaxGround: 3.0
hAngleRateCmdMax: 5.0 * pi / 6.0
hAngleCmdMax: 0.05


vVelocityMaxBody: 2.0
vVelocityMaxGround: 2.0
vPosCmdMax: 30.0
vPosCmdMin: 0.0
vThrustCmdMax: 1.0


yAngleRateMax: 1.0 #5.0 * pi / 6.0
yAngleMax: pi
